core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -58,13 +62,27 @@
    */
   @NonNull
   static DriverConfigLoader fromClasspath(@NonNull String resourceBaseName) {
+    return fromClasspath(resourceBaseName, DRIVER_CLASS_LOADER);
```

Comment:
Thinking about your comment [here](https://github.com/datastax/java-driver/pull/1410/files#r391900433): I wonder if this is the best class loader we can pass here and in other similar locations; maybe the context class loader is better because for web deployments it will cover more cases (and for OSGi, as you said, neither class loader would work anyways).